src/main/kotlin/app/api/SourcererApi.kt

```diff
@@ -0,0 +1,39 @@
+// Copyright 2017 Sourcerer Inc. All Rights Reserved.
+// Author: Anatoly Kislov (anatoly@sourcerer.io)
+
+package app.api
+
+import app.model.Commit
+import app.model.Repo
+import app.model.User
+
+/**
+ * Sourcerer API.
+ */
+object SourcererApi : Api {
+    var api: Api = ServerApi()
```

Comment:
Can you simply return a reference to api implementation if you want a singleton?

```
fun getInstance() { return api; }
```
and then you can delete all those methods that do precisely nothing.